### ISSUE TITLE
ubuntu 20 no virtualization tab

### ISSUE THREAD

**Issue by dongfanglong2012** [kimchi-project/wok#304]
I have installed wok kimchi and ginger-base after editing the .deb to have pyrhon3 where needed. I  kept finding in various places that doing so would fix the dependencies problem on ubuntu 20. The wok page does load just fine and there are no errors. I also changed the "/etc/kimchi/template.conf" file to point to images as mine is not default. I still have not virtualization tab though. I don't know what information would help but if there is anything you need to know let me know and ill try to provide it.

**Comment by dongfanglong2012** [comment]
found some more info from `sudo python3 /bin/wokd --environment=dev`
https://pastebin.com/mZzsa5S8

this line is odd to me. 
("Config error in section: 'wok', option: 'enable', value: 'True'. Config values must be valid Python.", 'TypeError',

because the file has `enable = True` and last i checked that is a valid command in python. i even tested it locally and it works. So im not sure what it means by valid python.

**Comment by x1nni** [comment]
+1 on this. Just installed on Ubuntu 20.04 and I am getting the exact same error.

**Comment by fisimania** [comment]
+1 on this. I have the same error on Ubuntu 20.04.

**Comment by nrdmtt** [comment]
Can you check /var/log/wok/wok-error.log to better understand the problem? I suspect it is something related to issue [1318](https://github.com/kimchi-project/kimchi/issues/1318)

**Comment by fisimania** [comment]
Log says:
Failed to load plugin conf from /etc/wok/plugins.d/kimchi.conf: ("Config error in section: 'wok', option: 'enable', value: 'True'. Config values must be valid Python.", 'TypeError', ("unrepr does not recognize 'Constant'",))

If I change function "def build(self, o):" in "class _Builder3:" to 
"def build(self, o):
        return o.value"
then kimchi plugin is loading, but noVNC is not working.

**Comment by nrdmtt** [comment]
Yep, that is another problem with ubuntu 20 :)  All should be fixed with the next release. 

If you are on a headless machine, see[ here](https://github.com/kimchi-project/kimchi/issues/1323#issuecomment-698899455) for a temporary workaround.
